packages/tofu/test/util.spec.js

```diff
@@ -1,7 +1,15 @@
 const test = require('ava')
 const { utils: { mergePolicy, mergeConfigEntire, objToMap, mapToObj } } = require('../src/index')
 
-testMergePartial('upgrades reads to writes', {
+const testMergePartial = test.macro(
```

Comment:
Nice! when was it introduced? I haven't seen test.macro used before.

Reply:
I have no idea, I'm new to `ava`, but discovered this while I was poking around.